src/test/java/com/softserve/edu/greencity/ui/pages/econews/EcoNewsPage.java

```diff
@@ -65,7 +65,7 @@
 
     public EcoNewsPage(WebDriver driver) {
         super(driver);
-        checkElements();
+        //checkElements();
```

Comment:
First it worked only without checkElements(), but then only with. We need to try again, what is better